### DISCUSSION TITLE
20231104周会纪要

### DISCUSSION THREAD

**Discussion by Mulavar** [arana-db/arana#797] [Weekly meeting]
## 1. 本周进展

| 功能点 | 参与人员 | 本周工作进展 | 备注 |
| --- | --- | --- | --- |
| [Feature] Support SQL join | @dongjh @sudu @pengjx |  | #600 <br> #622 |
| 1、[TX] support transaction XA  @chunshao  <br> ~~2、[XA] build auto clean transaction log ability @pangx~~ <br> ~~3、[XA] build transaction log ability @liy~~ <br> 4、[XA] build XA transaction back-and-forth decision-making  @haohf <br> 5、[tx] add deadlock for arana transaction. @liuyc  | @chunshao <br> @liuyc <br> @haohf <br> @liy（finish）<br> @pangx（finish） |  | #324  <br> #555 <br> #602 <br> #607 |
| shardingless feature | @yangwei |  |  |
| [Test] add unit test for /arana/pkg/runtime/ast/... package.  | @maronghe | #676  | #743 |
| [SQL] Support show table rules from xxx | @zhangbinbin |  | #628 |
| [Bug]Optimize show open tables xxx statment. | @huxiaohan | | #676 |
| [Test] add unit test for /arana/pkg/schema/... package. | @liyi  | | #729 |
| [Test] add unit test for /arana/pkg/trace/... package.  | @maorui | |  #741 |
| [Test] add unit test for /arana/pkg/runtime/optimize/... package. | @derecknowayback | | #747 |
| [Test] add unit test for /arana/pkg/misc/... package. | @linrl3 | | #745 |

## 2. 下周工作
| 功能点 | 参与人员 | 备注 |
| --- | --- | --- | 
| [Feature] Support SQL join | @dongjh @sudu @pengjx |  #600 <br> #622 |
| 1、[TX] support transaction XA  @chunshao  <br> ~~2、[XA] build auto clean transaction log ability @pangx~~ <br> ~~3、[XA] build transaction log ability @liy~~ <br> 4、[XA] build XA transaction back-and-forth decision-making  @haohf <br> 5、[tx] add deadlock for arana transaction. @liuyc  | @chunshao <br> @liuyc <br> @haohf <br> @liy（finish）<br> @pangx（finish） |  #324  <br> #555 <br> #602 <br> #607 |
| shardingless feature  | @yangwei |  |
| [Test] add unit test for /arana/pkg/runtime/ast/... package.  | @maronghe | #676  #743 |
| [SQL] Support show table rules from xxx | @zhangbinbin |  #628 |
| [Bug]Optimize show open tables xxx statment. | @huxiaohan | #676 |
| [Test] add unit test for /arana/pkg/schema/... package. | @liyi  | #729 |
| [Test] add unit test for /arana/pkg/trace/... package.  | @maorui | #741 |
| [Test] add unit test for /arana/pkg/runtime/optimize/... package. | @derecknowayback | #747 |
| [Test] add unit test for /arana/pkg/misc/... package. | @linrl3 | #745 |

**Comment by dongzl** [comment]
1. shardingless方案杨伟大佬先继续往细了出，shardingless分为创建、读写、扩缩容那个三个阶段，准备在扩缩容阶段对分段和哈希方案二选一
2. 毛锐那边的加单测的issue完成了，他后续跟着做分布式事务
3. huxiaohan那个bug fix的issue我弄到我这了，雨哥说他最近应该不来了
4. 后续需要补充下文档，比如快速入门啥的，我建了个简单的issue，后面可以讨论下像单测一样我们计划要写的文档让大家帮忙补充下，装点下门面
